docs/infrastructure/microk8s-quickstart.md

```diff
@@ -0,0 +1,325 @@
+
+## 5. Installing Codacy
+
+### 1. Helm and Tiller
+
+First we must install `helm` onto the cluster. With `helm`, we can easily install charts and manage the lifecycle of the installed artifacts. One example is the ability to rollback a failed install.
+
+1. Assuming you would want to install `helm` version 2.16.3, export the following environment variable
+    >export HELM_VERSION=2.16.3
+   1. Please note that we currently do *not* support helm v3.
+2. Run this script that will retrieve the desired `helm` version and install it onto the machine's `/usr/local/bin`:
+   1. Define a HELM_PKG variable
+      >HELM_PKG=helm-v${HELM_VERSION}-linux-amd64.tar.gz
+   2. Get the package
+      >wget https://get.helm.sh/$HELM_PKG
+   3. Extract the tarball
+      >tar xvzf $HELM_PKG
+   4. Move the binaries to `/usr/local/bin`
+      >mv linux-amd64/tiller linux-amd64/helm /usr/local/bin
+   5. Delete the extracted tarball directory
+      >rm -rvf $HELM_PKG linux-amd64/
+
+3. We must now deploy `tiller` onto the cluster.
+   1. First we must set a `serviceaccount` and a `clusterrolebinding` for `tiller` on the microk8s cluster:
+      >microk8s.kubectl create serviceaccount --namespace kube-system tiller
+
+      >microk8s.kubectl create clusterrolebinding tiller-cluster-rule --clusterrole=cluster-admin --serviceaccount=kube-system:tiller
+   1. Then we can install `tiller`, which will be installed with the same `HELM_VERSION` that you have specified earlier:
+      >helm init --service-account tiller
+   1. We wait for tiller to be ready
+      >microk8s.kubectl -n kube-system wait --for=condition=Ready pod -l name=tiller --timeout=300s
+
+### 2. Codacy chart
+
+Please refer to [our chart installation documentation](https://github.com/codacy/chart/blob/master/docs/install.md).
+
+Additionally, any `kubectl` command from [our chart installation](https://github.com/codacy/chart/blob/master/docs/install.md) must be executed as a `microk8s.kubectl` command.
+
+#### 1. Default Values
+
+We provide a base (values file)[##microk8s-values.yaml] that you can use for your microk8s Codacy installation. This file provisions a smaller number of component replicas with limited resources (i.e. CPU and memory). Should you want to use this as a template, please pass your `microk8s-values.yaml` file on (4.) of [our chart installation](https://github.com/codacy/chart/blob/master/docs/install.md). Lastly, it is important to correctly set your `codacy-api.ingress.hosts.host` if you are using this file as a template.
+
+#### 2. External Databases
+
+If you have decided to go with the [default values for the installation](####1.-Default-Values), your microk8s cluster will contain a PostgreSQL database within itself. This is not recommended as the data stored in the database will not be as durable if you store it in an external database.
+
+To setup an external database, you should follow the [steps described here](../requirements.md).
```

Comment:
I will leave that call for @pauloribeiro-codacy to make :) Just because we don't want to repeat ourselves across the docs so maybe we need to add that info elsewhere (not on microk8s docs)

Reply:
The initial version of the EKS documentation also mentioned a "built-in" database and then making the installation production-ready with an external one. But due to feedback we decided to simplify and include explicit instructions only on how to set up Codacy with the (recommended) external database. So, I suggest that we focus on the recommended path of setting up the connection to the external database that is already part of the pre-requirements. You can check with @paulopontesm regarding what was done for EKS.